dev-java/java-service-wrapper/java-service-wrapper-3.5.50.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+inherit java-pkg-2 java-ant-2 toolchain-funcs
+
+MY_PN="wrapper"
+MY_P="${MY_PN}_${PV}_src"
+DESCRIPTION="A wrapper that makes it possible to install a Java Application as daemon"
+HOMEPAGE="https://wrapper.tanukisoftware.com/"
+SRC_URI="https://${MY_PN}.tanukisoftware.com/download/${PV}/${MY_P}.tar.gz"
```

Comment:
why not have the non-variable parts plain-texted?
SRC_URI="https://download.tanukisoftware.com/wrapper/${PV}/wrapper_${PV}_src.tar.gz"

Reply:
This was just copied from the previous version.